libs/components/packages/src/schematics/ng-generate/add-ai-skills/tests/migration-resolver/test-behavioral-overlay.md

```
@@ -0,0 +1,68 @@
+
+    const confirmHarness = await loader.getHarness(SkyConfirmHarness);
+    const title = await confirmHarness.getMessageText();
+
+    expect(title).toBe('Are you sure you want to delete this record?');
+  });
+});
+```
+
+The error:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)